stable/kube-state-metrics/templates/clusterrole.yaml

```diff
@@ -177,4 +177,10 @@ rules:
     - verticalpodautoscalers
   verbs: ["list", "watch"]
 {{ end -}}
+{{ if .Values.podSecurityPolicy.enabled }}
+- apiGroups: ["extensions"]
```

Comment:
I think we should also allow access for the “policy” apiGroup. This could be achieved with a kubeVersion check.

Reply:
@tariq1890 If you think that it strongly belongs together, I can add it. However, I'd like you to elaborate on the idea. Afaiu [kubeVersion can be accessed in the template](https://helm.sh/docs/chart_template_guide/builtin_objects/) and specified in [Chart.yaml]. How do you want to use it. I think it's better to tackle this proposal in a separate PR since we need to exchange at least a bit.

Reply:
Take a look at:
https://github.com/helm/charts/blob/c160e0543d1d1b7097c661e776e2d1152ab2e6bf/stable/prometheus-operator/templates/prometheus-operator/admission-webhooks/job-patch/clusterrole.yaml#L22

Reply:
@mrueg Thanks for the link. I'm not sure whether this is right, please take a look.